python/lsst/meas/base/measurementInvestigationLib.py

```diff
@@ -0,0 +1,125 @@
+#
+# LSST Data Management System
+#
+# Copyright 2008-2017  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+from collections import Iterable
+
+from lsst.afw.table import SourceCatalog
+from lsst.meas.base import NoiseReplacer, NoiseReplacerConfig
+
+
+def rebuildNoiseReplacer(exposure, measCat):
```

Comment:
The reason they are in their own sub module is because they are a separate unit of work. They make use of NoiseReplacer, but the are not logically bound together any more than single frame measurement task should be in the same file as characterize image task. Up to this point we have not had a code for, or a place to put, functions that were used for examining measurements. I am hoping this can become that place.